homeassistant/strings.json

```
@@ -53,7 +53,8 @@
         "already_configured_device": "Device is already configured",
         "no_devices_found": "No devices found on the network",
         "oauth2_missing_configuration": "The component is not configured. Please follow the documentation.",
-        "oauth2_authorize_url_timeout": "Timeout generating authorize URL."
+        "oauth2_authorize_url_timeout": "Timeout generating authorize URL.",
+        "oauth2_no_url_available": "No URL available"
```

Can we link them to our website to go to get more information ? 
```suggestion
        "oauth2_no_url_available": "No URL available. For information about this error, [check the help section]({docs_url})"
```

And then do `self.async_abort(…, description_placeholders={'docs_url': 'https://…'})`

---

Pointed it to https://www.home-assistant.io/docs/configuration/basic/ for now, this explains how you set the URL's
But that does not explain much of why you get this message.

Should I create a new page for this, and if so where would it fit?